server/channels/api4/post_test.go

```diff
@@ -131,7 +131,7 @@ func TestCreatePost(t *testing.T) {
 		require.NoError(t, err)
 
 		// Message with no channel mentions should result in no ephemeral message
-		timeout := time.After(2 * time.Second)
+		timeout := time.After(5 * time.Second)
```

Comment:
Not for this PR, but looks like we are using this pattern over and over again, should we have something like `testutils.RequireEventuallyWebSocketEvent` and `testutils.RequireNeverWebSocketEvent` or something like that?

Reply:
Yes probably.